mysql-ballerina/client.bal

```diff
@@ -169,13 +169,12 @@ public type SSLMode SSL_PREFERRED|SSL_REQUIRED|SSL_VERIFY_CERT|SSL_VERIFY_IDENTI
 # SSL Configuration to be used when connecting to mysql server.
 #
 # + mode - `SSLMode` to be used during the connection
-# + clientCertKeystore - Keystore configuration of the client certificates
-# + trustCertKeystore - Keystore configuration of the trust certificates
-#
-public type SSLConfig record {|
+# + key - Keystore configuration of the client certificates
+# + cert - Keystore configuration of the trust certificates
+public type SecureSocket record {|
     SSLMode mode = SSL_PREFERRED;
-    crypto:KeyStore clientCertKeystore?;
-    crypto:KeyStore trustCertKeystore?;
+    crypto:KeyStore key?;
```

Comment:
Can't we make this mandatory?
```suggestion
    crypto:KeyStore key;
```

Reply:
we can not make that field mandatory. according to the mysql documentation, there are several ssl mode available and key and cert is depended on that mode selection.

Reply:
@lahirusamith Just out of curiosity, what is the mode (use case), where we have to use `cert` field without `key` field?

Reply:
both key and cert may be optional for SSL_PREFERRED.